### PR TITLE
[PROJECT] Clock balance & fixes

### PR DESCRIPTION
**Pull request by Zonespace27**
## About The Pull Request
Lots of balance and fixes for remaining stuff:

- APCs will fully drain if cogged, instead of bottoming out at 50%
- Tons of runtimes fixed
- Removed eye damage from wraith spectacles, because I felt like it contributed nothing and was very lame
- Fixed clockcult airlocks, now they shock you if you're a non-cultist
- Per-tile lock-on time for the clockwork rifle from 0.2s to 0.25s
- Added clockwork pickup element to clockwork rifle and brass bow.
- Marauder shield count from 4 to 5
- Marauders now have Whirring Convergence
- Fixes errant clockwork pickup message
- Clockwork armaments from 150W to 450W
- Ocular warden stats buffed, now requires power to work
- You can now hit clockwork structures with a slab to get more information on it
- Fixes icon states of a bunch of structures when powered/depowered
- Tinkerer's cache can now produce a full toolbelt of brass tools, along with traps
- Tinkerer's cache power cost of everything raised 2x
- Pressure sensor icon is now smaller
- Trap placer now works by hitting a tile
- Reworked clockwork slab information
- Clockwork slab now has a recall spell, all clock cultists gain it, works on latest-used slab

## How This Contributes To The Skyrat Roleplay Experience
Last clock cult PR before I'm calling this good-to-ship, I have more planned later, but I'd like this to be done

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
I tested this extensively

</details>

### CODE REVIEW

**Comment by Zonespace27** [comment]
This was live for my event and worked fine

File: modular_skyrat/modules/clock_cult/code/actions/recall_slab.dm [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+	desc = "Recall your latest used Clockwork Slab from anywhere in the universe."
+	button_icon_state = "Replicant"
+
+	///The slab marked for recall
+	var/obj/item/clockwork/clockwork_slab/marked_slab
+
+
+/// Set the passed object as our marked item
+/datum/action/innate/clockcult/recall_slab/proc/mark_item(obj/to_mark)
+	marked_slab = to_mark
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/clock_cult/code/actions/recall_slab.dm:12]: You might want to typecheck this

File: modular_skyrat/modules/clock_cult/code/structures/traps/trap.dm [hunk 2]
```diff
@@ -13,7 +13,24 @@
 	if(!IS_CLOCK(user))
 		return
 
-	for(var/obj/structure/destructible/clockwork/trap/trap in get_turf(src)) // No 50-spear instakills please
+	if(user.loc != get_turf(src))
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/clock_cult/code/structures/traps/trap.dm:16]:
```suggestion
	if(get_turf(user) != get_turf(src))
```

**Review by Gandalf2k15 — APPROVED** [review]
yee

File: modular_skyrat/modules/clock_cult/code/actions/recall_slab.dm (hunk 1 above)
Comment (Zonespace27) [reply modular_skyrat/modules/clock_cult/code/actions/recall_slab.dm:12]: it'll always be a slab

File: modular_skyrat/modules/clock_cult/code/structures/traps/trap.dm (hunk 2 above)
Comment (Zonespace27) [reply modular_skyrat/modules/clock_cult/code/structures/traps/trap.dm:16]: This was intended so that people don't lay traps while in a locker or w/e

**Pull request merged**